### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by shwilks**
Various small edits, main changes are that the map is hopefully now styled to hide tools other than the polygon one, and also user attempts to hit enter when searching for an address are ignored.

### CODE REVIEW

**Review by StephanieWillis — COMMENTED** [review]
This is wonderful Sam - thank you!

A couple of small things:
1. If you could modify the enter piece to work when someone presses enter that would be great. No worries if not. 

2. Because this is  a fork rather than a branch, it's not straightforward for me to pull this and run in locally. I dare say that is the normal flow though? Do you know what the standard practices is - should I just clone your repo so I can run it locally before merging?

**Comment by StephanieWillis** [comment]
Also: A couple of tests are failing because of the type change in Fuel. Probably not anything substantive but worth a quick look?

 FAILED tests/test_consumption.py::test_consumption_stream_annual_sum - TypeError: 'Fuel' object is not callable

File: src/place_search/index.html [hunk 1]
```diff
@@ -66,6 +66,14 @@
         };
         var autocomplete = new google.maps.places.Autocomplete(input, options);
 
+        // Interrupt the enter event so that the user has to select from place list
```
[the remark this answers is not in the document]
Comment (shwilks) [reply src/place_search/index.html:69]: ah I see, often the dropdown box is still displaying multiple potential matches though, but I guess you are suggesting that you basically just select the first match listed when you press enter? -cool by the way that it looks like you have made your own streamlit component for that bit.

**Comment by shwilks** [comment]
oops.. I was a bit lazy not running the tests locally, can take a look! :)

**Comment by shwilks** [comment]
> Because this is a fork rather than a branch, it's not straightforward for me to pull this and run in locally. I dare say that is the normal flow though? Do you know what the standard practices is - should I just clone your repo so I can run it locally before merging?

Yes it's a normal flow but not something I've done a lot of.  There seems to be some options using the github CLI tools (https://stackoverflow.com/questions/33814783/git-how-to-fetch-a-pull-request-for-testing) for fetching pull requests by PR number.

What I would normally do though is still simply run fetch though, it's no more difficult than fetching from your own fork branch, i.e.

```
git fetch https://github.com/shwilks/solar-and-heat-pump-savings.git small-edits
```

Which should fetch it as a local branch for you called small-edits that you can then checkout and test.

If we're going to be doing this a lot you can setup my repo as another remote called `shwilks` or something and save you pasting the whole repo address.  You can also checkout a new branch from your local master and then pull from my fork into that if you prefer.

If you then make some changes that fix things I think you can then simply push them back into my small-edits branch and this PR updates accordingly, or probably you can't push directly into my branch either and have to do that with a PR for that too which seems clunky - for that bit I get a bit hazy how it would work best?

Anyway would be curious to hear what the most normal way is though, what does Archy think?

File: src/roof.py [hunk 2]
```diff
@@ -155,6 +151,42 @@ def roof_mapper(width: int, height: int) -> Optional[List[Polygon]]:
...
+             "    a.leaflet-draw-toolbar-button-enabled, " \
+             "    a.leaflet-draw-toolbar-button-enabled:hover " \
+             "    { background-color: lightblue; }';" \
+             "    stylesheet.sheet.insertRule(styles, 0);" \
+             "    console.log('styling_map');" \
+             "  }" \
+             "});"
+
+    # Add the extra script you defined
+    streamlit_components.html(f"<script>{script}</script>", height=0)
```
[the remark this answers is not in the document]
Comment (shwilks) [reply src/roof.py:187]: :D thanks, it was a pain but learnt a lot more about the inner workings of streamlit.  Unfortunately their decision to nest everything in iframes seems like a bad one that makes this sort of thing quite ugly..

**Comment by shwilks** [comment]
ok, for now I just reverted the change I made to make it compatible with the latest python version which was causing problems with the tests.  Would like to understand better what's going on there though.

**Comment by shwilks** [comment]
and a final test change I needed locally to account for some floating point error differences

File: src/place_search/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (StephanieWillis) [reply src/place_search/index.html:69]: Finally played with the and its great - much better than telling people no address was found. Thank you.

**Review by StephanieWillis — APPROVED** [review]
Thank you so so much! And sorry for being so slow to merge. These changes massively improve the usability!

**Pull request merged**